src/main/java/netcracker/study/api/controller/IBootstrap.java

```diff
@@ -0,0 +1,11 @@
+package netcracker.study.api.controller;
+
+public interface IBootstrap {
+
+    void start();
+
+    String nextLine();
+
+    Integer nextInt();
```

Comment:
Bootstrap is a high-level entity and it should not offer low level operations like nextLine() and nextInt();